aplib.net-demo/Assets/Scripts/Items/Item.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Item : MonoBehaviour
+{
+
+    //This is where you change variables that are shared between all the items. The values themselves you want to change in the prefab for the specific item you want.
+    //To create a new item: Create a new script that inherits from this one, and then make a prefab that has that script attached.
+    public float uses;
+    public float startUses;
+    public string itemName;
```

Comment:
We are now repeating 'item', we are already creating an item, so it should just have a `name`.